README.md

```diff
@@ -1,10 +1,30 @@
 # Simplenote for iOS
 A Simplenote client for iOS. Learn more about Simplenote at [Simplenote.com](https://simplenote.com).
 
+
 ## Development Requirements
+
 * [CocoaPods](https://cocoapods.org/).
 * [Xcode](https://developer.apple.com/xcode/).
 
+
+## Setup Credentials
+
+Simplenote is powered by the [Simperium Sync'ing protocol](https://www.simperium.com). We distribute **testing credentials** that help us authenticate your application, and verify that the API calls being made are valid.
+
+In order to create your own Simperium.com Application, please, [head over to our website](https://www.simperium.com) and signup for free.
+
+After you've created your own Simperium application, copy the Sample Config as follows:
```

Comment:
I'd write this as:

> After you've created your own Simperium application, copy the sample config. For example:

```
cp Simplenote/config-demo.plist Simplenote/config.plist
```

(let's assume the user is in the top level dir of the git repo)